### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by no-u101**
should be all the bugs

### CODE REVIEW

File: js/core.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ window.onload = async () => {
     console.log("Loaded Quest");
 
     $("#quest_version").html(Quest.tag_name.replace("v", ""));
-    $("#quest_download_btn").attr("href", Quest.assets[0].browser_download_url);
+    $("#quest_download_btn").attr("href", Quest.assets[1].browser_download_url);
```
Comment (IMB11) [inline js/core.js:155]: This is perfectly fine. [0] retrieves the first file in the array (qmod) which is the latest release.

Arrays are index starting at 0, not 1. Change this please.

File: documentation.html [hunk 2]
```diff
@@ -62,7 +62,7 @@
         <div class="navbar-end">
             <div class="navbar-item">
                 <div class="buttons">
-                    <a class="button is-dark" href="#download">
+                    <a class="button is-dark" href="https://noodleextensions.com/#download">
```
Comment (IMB11) [inline documentation.html]: This causes a full site reload. Use /#download`

Comment (IMB11) [reply documentation.html]: And do it on 404 and report an issue page

**Review by IMB11 — CHANGES_REQUESTED** [review]

**Comment by no-u101** [comment]
done

**Review by IMB11 — APPROVED** [review]

**Pull request merged**